js/sykepengesoknad/felleskomponenter/sykmelding-utdrag/SykmeldingUtdrag.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { SykmeldingUtdrag as SykmeldingUtdragForArbeidstakere, sykmelding as sykmeldingPt } from '@navikt/digisyfo-npm';
+import { settErOppdelt } from '../../utils/settErOppdelt';
+import { ARBEIDSTAKERE, SELVSTENDIGE_OG_FRILANSERE } from '../../enums/soknadtyper';
+import SykmeldingUtdragForSelvstendige from '../../soknad-selvstendig-frilanser/sykmelding-utdrag/SykmeldingUtdragForSelvstendige';
+import { soknadPt } from '../../prop-types/soknadProptype';
+
+const Utdrag = ({ sykmelding, soknad, erApen, erOppdelt }) => {
+    return soknad.soknadstype === ARBEIDSTAKERE
+        && sykmelding
+        ? <SykmeldingUtdragForArbeidstakere
+            erApen={erApen}
+            sykmelding={sykmelding}
+            sykepengesoknad={{ _erOppdelt: erOppdelt }} />
+        : soknad.soknadstype === SELVSTENDIGE_OG_FRILANSERE
+            && sykmelding
+            ? <SykmeldingUtdragForSelvstendige
+                erApen={erApen}
+                sykmelding={sykmelding}
+                erOppdelt={erOppdelt} />
+            : null;
+};
+
+Utdrag.propTypes = {
+    sykmelding: sykmeldingPt,
+    soknad: soknadPt,
+    erApen: PropTypes.bool,
+    erOppdelt: PropTypes.bool,
+};
+
+const mapStateToProps = (state, ownProps) => {
+    const sykmelding = state.dineSykmeldinger.data.find((sykmld) => {
```

Comment:
kan du ikke bare skrive `sykmelding` helt ut?

Reply:
da får jeg en "sykmelding is already defined in upper scope"-feil av linteren... @knutnyg

Reply:
:(